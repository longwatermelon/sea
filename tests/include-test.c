#include "header.h"

fn main() -> int {
    let x:int = helper_function();
    let y:int = 5;
    return x + y;
}
