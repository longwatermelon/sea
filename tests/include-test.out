RETURN_CODE:47
STDOUT:
